kanidm_tools/src/cli/main.rs

```diff
@@ -21,6 +21,8 @@ fn main() {
             "RUST_LOG",
             "kanidm=debug,kanidm_client=debug,webauthn=debug",
         );
+    } else {
+        ::std::env::set_var("RUST_LOG", "kanidm=INFO,kanidm_client=INFO,webauthn=INFO");
```

Comment:
Since we have the tracing bits, I think I might rework this if that's okay?

Reply:
Of course 😄

Reply:
Hi @firstyear,

I do not know what you mean exactly.
 Are you suggesting to not put standard command output into a log level?

This is what I am wondering right now. Imagine some other cli tool, maybe even ls, I don't want to see timestamps and other potentially noisy stuff, until I ask for it. 
As you mention tracing, this is probably the right place for this type of information. But for a list of group members or groups queried by a user, I personally would like to get just this back, like in alpha6 and for other cli commands in the current version.

Let me know if I can help out somewhere.

Reply:
@cuberoot74088 No, I want to change how this if/else works, as there is a way in tracing to set default level filters that is a bit different.